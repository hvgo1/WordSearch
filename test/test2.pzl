RHPIRYUMAEA
HOALOIMSCTB
YMTYHAHIRNC
MOTRPUTMOOD
EPETAQYISME
YHRETPHLTAY
IONOEIREIIF
CNKPMCTICDG
SELBALLYSSH
CSSTELPUOCI
ABCDEYGHIJK

POETRY