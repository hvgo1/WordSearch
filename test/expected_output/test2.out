POETRY (4, 8) (4, 3)
